File: mission_1/question_4.py
#exo 4
marque = input("Marque de la voiture: ")
modèle = input("Modèle de la voiture: ")
prix_ht = int(input("Prix HT de l’unité: ")) #deamande de l'utilisateur

prix_TTC = prix_ht + 20/100 #calcul du prix avec taxe de 20 %

print(f"Votre{modèle}, de la marque {marque}, coûte {prix_TTC} €")

File: mission_1/question_4.cpp
#include <iostream>
using namespace std;

int main() {
  // Les questions
  int ht;
  cout << "Entrez le prix HT : ";
  cin >> ht;

  string marque;
  cout << "Entrez la marque de la voiture : ";
  cin >> marque;

  string modele;
  cout << "Entrez le modèle de la voiture : ";
  cin >> modele;

  // Calcul du prix TTC
  float ttc = ht + (0.2 * ht);

  cout << "Le modele " << modele << " de la marque " << marque << " coutera " << ttc << " euros" << endl;

  return 0;
}
